roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -13,17 +14,20 @@
   when:
     cephx and
     groups[restapi_group_name] is defined
+  run_once: true
```

Comment:
@erikh, not sure if this is a big one but I am wondering if this will prevent the key from generated on all monitor hosts (except one). Do we copy this key later to the monitor hosts? Otherwise we may not be able to make REST API call on monitor hosts other than the first one.